week-1/mandatory/2-classes-db/task.md

```diff
@@ -2,11 +2,19 @@
 
 ## Submission
 
-Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week.  You will find it beneficial to complete the reading tasks before attempting some of these.
+Below you will find a set of tasks for you to complete to consolidate and extend your learning from this week. You will find it beneficial to complete the reading tasks before attempting some of these.
 
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1.select * from rooms where rate>=100.00;
```

Comment:
Yes, a good solution but the question was "Which rooms have a rate of more than 100.00?", your solution includes rooms that have a rate = 100 (there are none, so you get the same results, but you need to be careful about the question).